tests/unittests/Foundation/NSNotificationQueueTests.mm

```diff
@@ -18,8 +18,10 @@
 #import <TestFramework.h>
 #import <Starboard/SmartTypes.h>
 #import <Helpers/TestHelpers.h>
+#import <Windows.h>
```

Comment:
yes. `Interlocked*` are from Windows.h, and none of the OS X platform headers include Windows.h for us